Dangerfile

```diff
@@ -0,0 +1,38 @@
+# Reference: http://danger.systems/reference.html
+
+# A pull request summary is required. Add a description of the pull request purpose.
+# Add labels to the pull request in github to identify the type of change. https://help.github.com/articles/applying-labels-to-issues-and-pull-requests/
+# Changelog must be updated for each pull request.
+# Warnings will be issued for:
+#    Pull request with more than 400 lines of code changed
+#    Pull reqest that change more than 5 lines without test changes
+
+def code_changes?
+  code = %w(libraries attributes recipes resources)
+  code.each do |location|
+    return true unless git.modified_files.grep(/#{location}/).empty?
+  end
+  false
+end
+
+def test_changes?
+  tests = %w(spec test .kitchen.yml .kitchen.dokken.yml)
+  tests.each do |location|
+    return true unless git.modified_files.grep(/#{location}/).empty?
+  end
+  false
+end
+
+raise 'Please provide a summary of your Pull Request.' if github.pr_body.length < 10
+
+warn 'This is a big Pull Request.' if git.lines_of_code > 400
+
+# Require a CHANGELOG entry for non-test changes.
+if !git.modified_files.include?('CHANGELOG.md') && code_changes?
```

Comment:
Looks like this should be `changelog.md`

Reply:
We should fix that, uppercase should/is the standard

Reply:
@Xorima You resolved these comments, but I don't see the commit that renamed the changelog file.  Just want to make sure that isn't missed